### PR TITLE
Update README with metrics

### PR DESCRIPTION
**Pull request by di-press**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,312 +1,323 @@
...
+
+|Class |Description             |
+| :--- | :---                   |
+|0     |No-op                   |
+|1     |Start of sentence       |
+|2     |Start of noise sequence |
+|3     |End of noise sequence   |
+
+
+In a curated dataset, comprised of ground-truth legal text segments, Ulysses Segmenter achieves higher Precision and Recall for the Class I (Segment) than other segmentation tools, such as NLTK, SpaCy and [LexNLP](https://github.com/LexPredict/lexpredict-lexnlp), with the latter being suitable for segmenting legal texts.
```
Comment (FelSiq) [inline README.md:17]: "Class I" -> "Class 1" is better

File: README.md [hunk 2]
```diff
@@ -1,312 +1,323 @@
...
+
+| Segmentation Method 	| Precision | Recall 	|
+|:---	    |:---	    |:---	    |
+| NLTK  	| 12.8557% 	| 19.6128% 	|
```
Comment (FelSiq) [inline README.md:21]: Could you either link NLTK, SpaCy, and LexNLP source directory or documentation in this table (or in line 17), please?

**Review by FelSiq — CHANGES_REQUESTED** [review]
Thanks @di-press . I've requested a few changes before merging into master.

File: README.md (hunk 1 above)
Comment (di-press) [reply README.md]: Ok! :)

File: README.md (hunk 2 above)
Comment (di-press) [reply README.md:21]: Done all the requested changes! :)

**Review by FelSiq — APPROVED** [review]

**Pull request merged**